lisa/sut_orchestrator/util/device_pool.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, List, Optional
+
+from lisa.sut_orchestrator.util.schema import HostDevicePoolSchema, HostDevicePoolType
+from lisa.util import LisaException
+
+
+class BaseDevicePool:
+    def _create_device_pool(
+        self,
+        pool_type: HostDevicePoolType,
+        vendor_id: str,
+        device_id: str,
+    ) -> None:
+        raise NotImplementedError()
+
+    def _get_primary_nic_id(self) -> List[str]:
+        raise NotImplementedError()
+
+    def _request_devices(
+        self,
+        pool_type: HostDevicePoolType,
+        count: int,
+    ) -> Any:
+        raise NotImplementedError()
+
+    def _release_devices(
```

Comment:
It's called out of BaseDevicePool, it should be public.

Reply:
Done